recipes-hulks/aliveness/aliveness.bb

```diff
@@ -5,185 +5,188 @@ LIC_FILES_CHKSUM = "file://${WORKDIR}/git/COPYING;md5=5b4473596678d62d9d83096273
 
 inherit cargo
 
-SRC_URI = "git://github.com/hulks/hulk;branch=main;protocol=https"
-SRCREV = "cce0d1af62f8126f6e51e648607d30d0c30a3657"
+SRC_URI = "git://github.com/julianschuler/hulk;branch=hula-dbus;protocol=https"
+SRCREV = "d5dbc8746e9a4d4af5a1412f389f855d93ca575f"
 S = "${WORKDIR}/git/tools/aliveness"
 
-inherit systemd
-
+SYSTEMD_PACKAGES = "${PN}"
```

Comment:
Why do you need this? This is the default https://docs.yoctoproject.org/singleindex.html#term-SYSTEMD_PACKAGES

Reply:
It was present in `hula.bb`, `hulk.bb` and `network-config.bb`, so I added this here as well. I removed these lines now in all files.